patches/juju_utils-add-os-series.diff

```diff
@@ -0,0 +1,48 @@
+This is the diff for https://github.com/juju/utils/pull/297. Can't build juju against master of utils for the moment
+
+diff --git a/github.com/juju/utils/series/supportedseries.go b/github.com/juju/utils/series/supportedseries.go
+index 70c17b8..fef97de 100644
+--- a/github.com/juju/utils/series/supportedseries.go
```

Comment:
before we actually land this, we'll want to land the other code in utils, and then just update dependencies.tsv as part of this patch to depend on the updated version.